llvm/test/TableGen/generic-tables.td

```diff
@@ -24,34 +24,35 @@ include "llvm/TableGen/SearchableTable.td"
 
 // CHECK-LABEL: GET_ATable_IMPL
 // CHECK: constexpr AEntry ATable[] = {
-// CHECK-NOT:   { "aaa"
-// CHECK:       { "baz"
-// CHECK:       { "foo"
-// CHECK:       { "foobar"
-// CHECK:       { "bar"
+// CHECK-NOT:   { "aaa", 0x0, 0x0, 0x0 },
```

Comment:
```suggestion
// CHECK-NOT:   { "aaa"
```

Reply:
Being a negative check, I'd live it as it is. Otherwise, if you introduce a bug that generates something slightly different, say `{ "aaa", 0x0, 0x0, 0x1 },`, you won't be able to prevent the regression. Essentially the `CHECK-NOT` is checking `isNeeded=0`, not the specific values passed to the `AEntry` class.

Reply:
Thanks! Applied!